app/models/setting/auth.rb

```diff
@@ -24,6 +24,7 @@ def self.load_defaults
         self.set('ssl_priv_key', N_("SSL Private Key file that Foreman will use to communicate with its proxies"), ssl_priv_key, N_('SSL private key')),
         self.set('ssl_client_dn_env', N_('Environment variable containing the subject DN from a client SSL certificate'), 'SSL_CLIENT_S_DN', N_('SSL client DN env')),
         self.set('ssl_client_verify_env', N_('Environment variable containing the verification status of a client SSL certificate'), 'SSL_CLIENT_VERIFY', N_('SSL client verify env')),
+        self.set('ssl_client_cert_env', N_('Environment variable containing a client\'s SSL certificate'), 'SSL_CLIENT_CERT', N_('SSL client cert env')),
```

Comment:
Can you use double quotes here?  We use a code parser to extract strings for translation, and it might be safer than escape characters.. it's pretty dumb.